doc/Extensions/OpenTSDB.md

```diff
@@ -0,0 +1,27 @@
+source: Extensions/OpenTSDB.md
+# Enabling support for OpenTSDB.
+
+This module sends all metrics to OpenTSDB server. You need something like Grafana for graphing.
+
+### Requirements
+ - OpenTSDB
+ - Grafana
+ 
+### What you don't get
+ - Pretty graphs, this is why at present you need Grafana. You need to build your own graphs within Grafana.
+ 
+RRD will continue to function normally so LibreNMS itself should continue to function normally.
+
+configuration in /opt/librenms/includes/defaults.inc.php.
```

Comment:
You have already said to put it the configuration in defaults.inc.php the last time?? when i will put it config or defaults ??? or the two ? 
![put](https://user-images.githubusercontent.com/28858069/28165086-8f10d044-67d2-11e7-811f-f157dccd8b09.PNG)